web/gds-user-ui/src/utils/axios.ts

```diff
@@ -47,10 +47,10 @@ axiosInstance.interceptors.response.use(
     return response;
   },
   async (error: any) => {
-    // let _retry = 0;
     const originalRequest = error?.config;
-    originalRequest._retry = originalRequest?._retry || 0;
-    //
+    if (originalRequest) {
```

Comment:
This will hopefully resolve an undefined error since adding optional chaining `(?.)` didn't work.